elements/lisk-p2p/test/functional/outbound_eviction.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright © 2019 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { expect } from 'chai';
+import { P2P } from '../../src/index';
+import { wait } from '../utils/helpers';
+import { platform } from 'os';
+
+describe('Network with frequent outbound peer shuffling', () => {
```

Comment:
Can we merge the two files related into a single file named `eviction`?

Reply:
I would rather keep these separate as they are using separate configurations and the internal mechanics are quite different.